packages/scandipwa/src/component/ProductAttributes/ProductAttributes.container.js

```diff
@@ -33,11 +33,15 @@ export class ProductAttributesContainer extends PureComponent {
     }
 
     _getAttributesWithValues() {
-        const { product: { attributes = {}, parameters = {} } } = this.props;
+        const { product: { attributes = {}, parameters = {}, name } } = this.props;
 
         const allAttribsWithValues = Object.entries(attributes).reduce((acc, [key, val]) => {
             const { attribute_label, attribute_value } = val;
 
+            if (key === 'name') {
+                return { ...acc, [attribute_label]: { ...val, attribute_value: name } };
+            }
+
```

Comment:
We getting this attribute value from Backend and in case if it is not correct, better to handle it there so on Frontend we will be getting correct value from beginning and no additional checks will be needed here.